### PR TITLE
Updating for tutum

### PR DESCRIPTION
**Pull request by moxiegirl**
Changes in support of the tutum

Signed-off-by: Mary Anthony <mary@docker.com>

### CODE REVIEW

**Comment by moxiegirl** [comment]
@thaJeztah @SvenDowideit Wanna take a look and review guys.  This is to support the GA of Docker Cloud in Jan.

File: layouts/partials/cloud-navbar.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<section id="multiple" class="tutmenu" data-accordion-group>
```
Comment (thaJeztah) [inline layouts/partials/cloud-navbar.html]: is this the right classname?

File: layouts/docker-cloud/single.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{{ partial "header.html" . }}
+<div class="row">
+    <div class="large-3 columns" class="tutmenu">
```
Comment (thaJeztah) [inline layouts/docker-cloud/single.html]: same here?

**Comment by thaJeztah** [comment]
One question, but LGTM

File: layouts/partials/cloud-navbar.html (hunk 1 above)
Comment (moxiegirl) [inline layouts/partials/cloud-navbar.html]: Yep.  I'm using the tutorial for these.

File: layouts/docker-cloud/single.html (hunk 2 above)
Comment (moxiegirl) [inline layouts/docker-cloud/single.html]: yep.

**Comment by moxiegirl** [comment]
author LGTM.  Thanks for the thoughtful eye Seb.

**Pull request merged**